data/curated/wa-cannabis/intro.md

```diff
@@ -0,0 +1,47 @@
+🌿 The Washington Cannabis Industry: Inventory and Sales Reporting (2024–2025)
+
+This week, we’re exploring Washington State’s legal cannabis market through a dataset of retail transactions and inventory classifications reported by licensed businesses!
+
+Since the legalization of recreational cannabis in Washington, the Washington State Liquor and Cannabis Board (WSLCB) has mandated that all licensed producers, processors, and retailers report detailed inventory movements and sales. These reports are used for regulatory compliance, taxation, and market analysis, and are made available to the public through open data initiatives.
+
+This dataset is a cleaned, anonymized sample of 50,000 retail sales transactions from January 2024 through March 2025. It is a representative subset drawn from over 19 million records in the original dataset (approximately 11.6 GB in size).
+
+The monthly aggregated data includes information on:
+
+Product type and strain
+
+Sale date (month-year)  
+
+Retail price, quantity sold, and taxes
+
+Basic licensee, licensee-to-licensee, and retailer-to-consumer transactions (when sold_to_licensee_dba = NA, this NA represents the anonomyous public consumer in a transaction with a licensed marijuana retailer or licensee_dba). 
+
+Inventory categories and subcategories (e.g., flower, concentrate, edibles)
+
+Transformed variables:
+
+price_gram_wavg = mean(calc_price_per_gram, weighted by sales_qty)
+
+price_absolute_diff = abs(calc_price_per_gram - price_gram_wavg)  
+
+price_percent_diff = 100 * (calc_price_per_gram - price_gram_wavg) / price_gram_wavg
+
+In this version, personally identifiable licensee information or PII has been removed, retaining only the business “Doing Business As” (DBA) name for analysis purposes.
+
+Participants this week can explore trends in cannabis pricing, product popularity, retail behaviors, and market structure over time. This dataset is righ-sized for piping directly into a Bayesian hierarchical model or forecasting pipeline.  
+
+Example questions to explore:
+
+How do sales volumes and prices change over time?
+
+Are there seasonal trends in retail cannabis activity? 
+
+What categories of cannabis products are most frequently sold?
+
+Does strain and product name interact with sales quantity or retail price?
+
+How does product pack size relate to price per gram?
+
+Do different licensee types or retail chains show pricing patterns?
+
+This dataset is ideal for time series forecasting, market basket analysis, text analysis, price modeling, and policy evaluation related to cannabis regulation.
```

Comment:
```suggestion
```
Work anything you want to keep from this sentence into line 31 above ("Participants this week can explore trends in cannabis pricing, product popularity, retail behaviors, and market structure over time. This dataset is right-sized for piping directly into a Bayesian hierarchical model or forecasting pipeline.").